documentation/decisions/adr/2023-12-15-deliverable-reporting-strategy.md

```diff
@@ -0,0 +1,235 @@
+  - Easy to parse in custom reporting.
+- **Cons**
+  - Does not support filtering or searching within the GitHub repository.
+  - Does not support filtering or searching within GitHub projects (e.g. roadmap or sprint board).
+  - Does not support grouping within GitHub projects or GitHub insight reporting.
+  - Very difficult to audit or maintain programmatically.
+
+## Evaluation - Limiting 30k deliverables in reporting <!-- OPTIONAL -->
+
+### Label
+
+This option involves creating a label that indicates when a deliverable should be included in reporting (e.g. "add to report") and then filtering for this label in our custom reporting.
+
+> [!NOTE]
+> The code should be configured to accept an arbitrary set of labels as filters, so that if the label changes, we only need to change the top-level configuration and not several lines of the source code.
+
+> [!TIP]
+> **Bottom line:** This is the best option if we:
+> - want an easy way to explicitly indicate when a deliverable should be included or excluded from reporting
+> - and are okay with decoupling the reporting logic from the business logic around deliverable status
+
+- **Pros**
+  - Makes the choice to include a deliverable in a report explicit and easy to understand.
+  - Enables users to add or remove deliverables from reporting without changing the logic or the source code.
+  - Makes it easy to change the logic that determines which issue labels are used to include/exclude issues from reporting.
+- **Cons**
+  - Decouples reporting logic from the status of ongoing deliverables. **Note:** this could be a pro or a con depending on how tightly coupled we want reporting and delivery management to be.
+  - Adds to the already long list of labels in the repository.
+
+### Status
+
+This option involves filtering the deliverables that are included in reporting based on their status in the provisional roadmap project. For example, we may only want to report on deliverables that have the status "In progress" or "Planning".
+
+> [!NOTE]
+> The code should be configured to accept an arbitrary set of statuses as filters, so that if we want to change which statuses are included, we only need to change the top-level configuration and not several lines of the source code.
+
+> [!TIP]
+> **Bottom line:** This is the best option if we:
+> - want deliverables to automatically be included in reporting once they enter a specific status (e.g. "in progress")
+> - and we don't anticipate edge cases that require including or excluding deliverables from reporting despite their status
+
+- **Pros**
+  - Enables users to add or remove deliverables from reporting without changing the logic or the source code.
+  - Makes it easy to change the logic that determines which statuses are used to include/exclude issues from reporting.
+  - Keeps the reporting and delivery management in sync.
```

Comment:
Maybe to expand on this: The reporting should, in theory, accurately update to focus reports automatically on the work we're doing now. (Which is great!)

Reply:
Expanded on this in this commit: [fix: Incorporates PR comments](https://github.com/HHS/simpler-grants-gov/pull/926/commits/5831bfabf360fca818dc281ddd31129b6152a7bd)